BlockServer/core/inactive_config_holder.py

```diff
@@ -98,6 +98,8 @@ def set_config_details(self, details):
                 for args in details["components"]:
                     comp = self.load_configuration(args['name'], True)
                     self.add_component(comp.get_name(), comp)
+            if "isProtected" in details:
+                self._config.meta.isProtected = details["isProtected"]
```

Comment:
In the schema, `isProtected` is a non-optional element (`minOccurs="1" maxOccurs="1"`). It should either be optional in the schema (recommended) or not optional here.

Reply:
Changed `isProtected` to an optional element